### DISCUSSION TITLE
error with alpinejs + Swiperjs

### DISCUSSION THREAD

**Discussion by slopesweb** [alpinejs/alpine#3365] [1. Help]
I'm trying to consume a wordpress json with alpine js + Swiper Js. But there is an error that I can not identify.

https://codepen.io/johnquimera/pen/mdjPvMO

Can someone help me?

**Comment by ekwoka** [comment]
What's the error?

**Comment by slopesweb** [comment]
![This is an image](https://imgur.com/a/LwJyhhb)
